website/docs/r/subnet.html.markdown

```diff
@@ -23,86 +23,97 @@ Class - fvSubnet
 ## Example Usage
 
 ```hcl
-	resource "aci_subnet" "foosubnet" {
-		parent_dn 		 = aci_bridge_domain.bd_for_subnet.id
-		description      = "subnet"
-		ip               = "10.0.3.28/27"
-		annotation       = "tag_subnet"
-		ctrl             = ["querier", "nd"]
-		name_alias       = "alias_subnet"
-		preferred        = "no"
-		scope            = ["private", "shared"]
-		virtual          = "yes"
-	}
-
-	# Create EP Reachability - Under AP -> EPG Subnet
-	resource "aci_subnet" "foo_epg_subnet_next_hop_addr" {
-		parent_dn     = aci_application_epg.foo_epg.id
-		ip            = "10.0.3.29/32"
-		scope         = ["private"]
-		description   = "This subject is created by terraform"
-		ctrl          = ["no-default-gateway"]
-		preferred     = "no"
-		virtual       = "yes"
-		next_hop_addr = "10.0.3.30"
-	}
-
-	# Create Anycast MAC - Under AP -> EPG Subnet
-	resource "aci_subnet" "foo_epg_subnet_anycast_mac" {
-		parent_dn   = aci_application_epg.foo_epg.id
-		ip          = "10.0.3.29/32"
-		scope       = ["private"]
-		description = "This subject is created by terraform"
-		ctrl        = ["no-default-gateway"]
-		preferred   = "no"
-		virtual     = "yes"
-		anycast_mac = "F0:1F:20:34:89:AB"
-	}
-
-	# Create MSNLB in IGMP mode - Under AP -> EPG Subnet
-	resource "aci_subnet" "foo_epg_subnet_msnlb_mcast_igmp" {
-		parent_dn   = aci_application_epg.foo_epg.id
-		ip          = "10.0.3.29/32"
-		scope       = ["private"]
-		description = "This subject is created by terraform"
-		ctrl        = ["no-default-gateway"]
-		preferred   = "no"
-		virtual     = "yes"
-		msnlb {
-			mode  = "mode-mcast-igmp"
-			group = "224.0.0.1" # The valid Multicast Address are 224.0.0.0 through 239.255.255.255.
-		}
-	}
-
-	# Create MSNLB in static multicast mode - Under AP -> EPG Subnet
-	resource "aci_subnet" "foo_epg_subnet_msnlb_mcast_static" {
-		parent_dn   = aci_application_epg.foo_epg.id
-		ip          = "10.0.3.29/32"
-		scope       = ["private"]
-		description = "This subject is created by terraform"
-		ctrl        = ["no-default-gateway"]
-		preferred   = "no"
-		virtual     = "yes"
-		msnlb {
-			mode  = "mode-mcast--static"
-			mac   = "03:1F:20:34:89:AA"
-		}
-	}
-
-	# Create MSNLB in unicast mode - Under AP -> EPG Subnet
-	resource "aci_subnet" "foo_epg_subnet_msnlb_mode_uc" {
-		parent_dn   = aci_application_epg.foo_epg.id
-		ip          = "10.0.3.29/32"
-		scope       = ["private"]
-		description = "This subject is created by terraform"
-		ctrl        = ["no-default-gateway"]
-		preferred   = "no"
-		virtual     = "yes"
-		msnlb {
-			mode  = "mode-uc"
-			mac   = "00:1F:20:34:89:AA"
-		}
-	}
+resource "aci_subnet" "foosubnet" {
```

Comment:
Can you also update the subnet.tf example in the examples/subnet directory to include ip data plane learning?